mods/persistence/modules/trade_beacons/trade_beacons.dm

```diff
@@ -0,0 +1,103 @@
+/atom/proc/get_adjacent_trade_beacons()
```

Comment:
Any reasons why this is an /atom proc?
If you don't override it, you could probably make a global proc out of it.

Reply:
it could potentially be overridden to allow for detecting trade beacons at a higher range. It's an atom proc because any entity with a loccould  look for adjacent trade beacons, both obj and turf. i'd like to keep this as is unless it's a requirement.